packages/@sanity/cli/templates/shopify/components/inputs/CollectionHidden.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import {WarningOutlineIcon} from '@sanity/icons'
+import {StringFieldProps} from 'sanity'
+import {Box, Card, Flex, Stack, Text} from '@sanity/ui'
+
+export default function CollectionHiddenInput(props: StringFieldProps) {
+  return (
+    <Card padding={4} radius={2} shadow={1} tone="critical">
+      <Flex align="flex-start">
+        <Text size={2}>
```

Comment:
Nitckpick: `size={2}` is the default text size, so it is not really necessary to use here